### DISCUSSION TITLE
What is a good approach to deal with breaking upgrades?

### DISCUSSION THREAD

**Discussion by oliversalzburg** [renovatebot/renovate#25877] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

I have this breaking change dependency upgrade in one of my packages. I've been working on resolving the breaking changes upstream and in the project, but it just takes a lot of time.

Meanwhile, the breaking upgrade is continuously rebased on top of whatever changes I make in the repo. This is exactly as configured, so no complaints here.

However, I was wondering how to deal with this more elegantly. I was searching for a Renovate setting to only rebase on a schedule, scale down rebasing attempts for failing upgrades, or something like that. Couldn't find anything though. So my inbox regularly looks like this:

![image](https://github.com/renovatebot/renovate/assets/1658949/2eefa8a0-e63e-46fc-bbf3-7b56cf748bc9)

I guess I could just close the PR and handle it manually once I'm able to resolve the conflicts, but I like to have this item on my "dashboard" so I don't forget eventually ugprading it.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
#20940 would help

**Reply by oliversalzburg** [reply]
Oh yeah, that seems nice.
